beets/logging.py

```diff
@@ -83,40 +91,45 @@ class StrFormatLogger(Logger):
     """
 
     class _LogMessage:
-        def __init__(self, msg, args, kwargs):
+        def __init__(
+            self,
+            msg: str,
+            args: logging._ArgsType,
+            kwargs: dict[str, Any],
+        ):
             self.msg = msg
             self.args = args
             self.kwargs = kwargs
 
         def __str__(self):
-            args = [logsafe(a) for a in self.args]
-            kwargs = {k: logsafe(v) for (k, v) in self.kwargs.items()}
+            args = [_logsafe(a) for a in self.args]
+            kwargs = {k: _logsafe(v) for (k, v) in self.kwargs.items()}
             return self.msg.format(*args, **kwargs)
 
     def _log(
         self,
-        level,
-        msg,
-        args,
-        exc_info=None,
-        extra=None,
-        stack_info=False,
+        level: int,
+        msg: object,
+        args: logging._ArgsType = (),
+        exc_info: logging._ExcInfoType = None,
```

Comment:
`_ArgsType` and `_ExcInfoType` are not intended to be used outside of `logging` module (original 😅) implementation.

My IDE does not jump into them nor resolve their definitions, so they are a mystery to me. Can you define them explicitly in this module, please?